frontend/src/lib/components/ReviewServiceRequest.svelte

```diff
@@ -15,7 +15,7 @@
 			</p>
 
 			<div class="my-4">
-				<ServiceRequestDetails serviceRequest={data} />
+				<ServiceRequestDetails serviceRequest={params} />
```

Comment:
does this component allow the CreateServiceRequestParams type as a prop?

Reply:
like this?
```
// ServiceRequestDetails.svelte
export let serviceRequest: ServiceRequest | Readonly<Partial<CreateServiceRequestParams>>;
```